sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -578,9 +578,6 @@ object AlluxioUtils extends Logging {
       // we know then fall back to inferring. The latter happens on certain CSPs
       // like Databricks where they have customer file index types.
       relation.location match {
-        case pfi: PartitioningAwareFileIndex =>
-          logDebug("Handling PartitioningAwareFileIndex")
-          createNewFileIndexWithPathsReplaced(pfi.partitionSpec(), pfi.rootPaths)
         case cfi: CatalogFileIndex =>
```

Comment:
I'm assuming same condition can happen here since we use the same FileIndex replacement

Reply:
The information of `CatalogFileIndex` is from catalog table.
Refer to the code: [Link1](https://github.com/apache/spark/blob/v3.3.0/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/CatalogFileIndex.scala#L73)

``` scala
val selectedPartitions = ExternalCatalogUtils.listPartitionsByFilter(
        sparkSession.sessionState.conf, sparkSession.sessionState.catalog, table, filters)
```

If we create table before executing a query, then the table will be refreshed.
If do refresh operation on this table, it also refresh.

IMO, we can keep it as it is now.